src/main/java/io/github/fablabsmc/fablabs/impl/fiber/annotation/AnnotatedSettingsImpl.java

```diff
@@ -264,60 +269,64 @@ public ConfigBranch makeTree(Object pojo) throws FiberException {
 		@SuppressWarnings("unchecked")
 		Class<P> pojoClass = (Class<P>) pojo.getClass();
 
-		boolean onlyAnnotated;
 		SettingNamingConvention convention;
 
 		if (pojoClass.isAnnotationPresent(Settings.class)) {
 			Settings settingsAnnotation = pojoClass.getAnnotation(Settings.class);
-			onlyAnnotated = settingsAnnotation.onlyAnnotated();
 			convention = createConvention(settingsAnnotation.namingConvention());
 		} else { // Assume defaults
-			onlyAnnotated = false;
 			convention = new NoNamingConvention();
 		}
 
-		NodeOperations.moveChildren(this.constructNode(pojoClass, pojo, onlyAnnotated, convention), mergeTo);
+		NodeOperations.moveChildren(this.constructNode(pojoClass, pojo, convention), mergeTo);
 	}
 
-	private <P> ConfigTreeBuilder constructNode(Class<P> pojoClass, P pojo, boolean onlyAnnotated, SettingNamingConvention convention) throws FiberException {
+	private <P> ConfigTreeBuilder constructNode(Class<P> pojoClass, P pojo, SettingNamingConvention convention) throws FiberException {
 		ConfigTreeBuilder node = ConfigTree.builder();
 
 		List<Member> defaultEmpty = new ArrayList<>();
-		Map<String, List<Member>> listenerMap = this.findListeners(pojoClass);
+		Map<String, List<Member>> listenerMap = new HashMap<>();
 
-		for (Field field : pojoClass.getDeclaredFields()) {
-			if (field.isSynthetic() || !isIncluded(field, onlyAnnotated)) continue;
+		ListenerProcessor listenerProcessor = new ListenerProcessor() {
```

Comment:
I would prefer implementing those interfaces directly on `AnnotatedSettingsImpl` rather than using anonymous classes.

Reply:
Problem is access to `listenerMap`, `convention`, `node` and `pojo` - do we want to be storing all of those as class members?